docs/GoogleDrive-batchsink.md

```diff
@@ -39,8 +39,16 @@ OAuth2 and Service account types are available.
 
 #### Service account properties
 
-**Account file path:** Path on the local file system of the service account key used for authorization.
-Can be set to 'auto-detect' when running on a Dataproc cluster. 
-When running on other clusters, the file must be present on every node in the cluster.
+**Account file path:** Path on the local file system of the service account key used for authorization. 
+Can be set to 'auto-detect' for getting service account from system variable.
+The file/system variable must be present on every node in the cluster.
 Service account json can be generated on Google Cloud 
-[Service Account page](https://console.cloud.google.com/iam-admin/serviceaccounts)
\ No newline at end of file
+[Service Account page](https://console.cloud.google.com/iam-admin/serviceaccounts)
+
+### Retrying
+
+**Max retry count:** Maximum number of retry attempts.
```

Comment:
upper case all words in all config names.

Reply:
done